src/core/public/chrome/ui/header/home_icon.tsx

```diff
@@ -4,51 +4,46 @@
  */
 
 import React from 'react';
-import { EuiIcon } from '@elastic/eui';
+import { EuiIcon, IconSize } from '@elastic/eui';
 import { ChromeBranding } from '../../chrome_service';
+import { Logos } from '../../../../common/types';
 
-export const DEFAULT_MARK = 'opensearch_mark_default_mode.svg';
-export const DEFAULT_DARK_MARK = 'opensearch_mark_dark_mode.svg';
+interface Props {
+  branding: ChromeBranding;
+  logos: Logos;
+}
 
 /**
  * Use branding configurations to render the header mark on the nav bar.
- *
- * @param {ChromeBranding} - branding object consist of mark, darkmode selection, asset path and title
- * @returns Mark component which is going to be rendered on the main page header bar.
  */
-export const HomeIcon = ({
-  darkMode,
-  assetFolderUrl = '',
-  mark,
-  applicationTitle = 'opensearch dashboards',
-  useExpandedHeader = true,
-}: ChromeBranding) => {
```

Comment:
Pretty significant change to API, this isn't public right?

Reply:
No :)
ref: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/index.ts#L51